roadmap.md

```diff
@@ -111,7 +110,15 @@ Mac OS X                      | 10.12+                        | x64           |
 * [S] This is supported by Samsung.
 * [C] This is supported by the Community
 
-Version 1.0 OS Support:
+### Unsuported OS Versions
```

Comment:
Thanks! I actually changed it to `Community-support`, which IMO better captures the real status.
I also did minor text tweaks, can you please check them Dan? Thanks!